crates/types/src/interpolated.rs

```diff
@@ -0,0 +1,68 @@
+use nalgebra::{matrix, point, Point2};
+use serde::{Deserialize, Serialize};
+use serialize_hierarchy::SerializeHierarchy;
+
+#[derive(Clone, Copy, Debug, Default, Deserialize, Serialize, SerializeHierarchy)]
+pub struct Interpolated {
+    pub first_half_own_half_towards_own_goal: f32,
+    pub first_half_own_half_away_own_goal: f32,
+    pub first_half_opponent_half_towards_own_goal: f32,
+    pub first_half_opponent_half_away_own_goal: f32,
+}
```

Comment:
I don't like the name `Interpolated`, but let's kloppen about that after the game.

Reply:
Sure, which year?